src/components/Home/AmountInput.js

```diff
@@ -18,9 +18,7 @@ export default function AmountInput({ value, error, maxLength, onChange, tintCol
 					fontSize: 70,
 					fontWeight: 'bold',
 					color: error == null ? tintColor : colors.error,
-					alignSelf: 'center',
 					textAlign: 'right',
-					width: value ? 'auto' : 180,
```

Comment:
Here it is required with Android or the last 0 is truncated.... Can you set 200 or more so it shows on iOS correctly too ?

Reply:
Should be fixed by reducing font size